### PR TITLE
es: Format /learn/javascript using Prettier

### PR DESCRIPTION
**Pull request by queengooborg**
This PR formats the `/learn/javascript` folder of the Spanish locale using Prettier.

### CODE REVIEW

File: files/es/learn/javascript/objects/basics/index.md [hunk 1]
```diff
@@ -33,28 +33,39 @@ Si ingresas `persona` en tu entrada de texto y presionas el botón, debes obtene
...
+        "" +
+        this.nombre[1] +
+        " tiene " +
+        this.edad +
+        " años. Le gusta " +
+        this.intereses[0] +
+        " y " +
+        this.intereses[1] +
+        ".",
+    );
```
Comment (Graywolf9) [inline files/es/learn/javascript/objects/basics/index.md:52]: using backquotes as `en-US`

```suggestion
    alert(`${this.nombre[0]} ${this.nombre[1]} tiene ${this.edad} años. Le gusta ${this.intereses[0]} y this.intereses[1].`);
```

File: files/es/learn/javascript/objects/object_building_practice/index.md [hunk 2]
```diff
@@ -184,21 +184,27 @@ Todos los programas que generan animaciones normalmente tienen un bucle de anima
...
+      random(0 + size, height - size),
+      random(-7, 7),
+      random(-7, 7),
+      "rgb(" +
+        random(0, 255) +
+        "," +
+        random(0, 255) +
+        "," +
+        random(0, 255) +
+        ")",
```
Comment (Graywolf9) [inline files/es/learn/javascript/objects/object_building_practice/index.md:206]: Taking from `en-US`

```suggestion
      `rgb(${random(0, 255)},${random(0, 255)},${random(0, 255)})`,
```

File: files/es/learn/javascript/objects/object_building_practice/index.md [hunk 3]
```diff
@@ -233,34 +239,41 @@ Ahora, un poco de diversión, añadamos la detección de colisiones a nuestro c
...
+
+         if (distance < this.size + balls[j].size) {
+           balls[j].color = this.color =
+             "rgb(" +
+             random(0, 255) +
+             "," +
+             random(0, 255) +
+             "," +
+             random(0, 255) +
+             ")";
```
Comment (Graywolf9) [inline files/es/learn/javascript/objects/object_building_practice/index.md:258]:
```suggestion
             `rgb(${random(0, 255)},${random(0, 255)},${random(0, 255)})`;
```

**Review by Graywolf9 — COMMENTED** [review]
Thank you so much! There are some comments

File: files/es/learn/javascript/building_blocks/return_values/index.md [hunk 4]
```diff
@@ -88,45 +88,55 @@ Let's have a go at writing our own functions featuring return values.
...
+         num +
+         " cubed is " +
+         cubed(num) +
+         ". " +
+         num +
+         " factorial is " +
+         factorial(num) +
+         ".";
+     }
+   };
```
[the remark this answers is not in the document]
Comment (queengooborg) [reply files/es/learn/javascript/building_blocks/return_values/index.md:134]: Ooh, much better looking!

**Review by Graywolf9 — APPROVED** [review]

**Pull request merged**